src/components/Chips/__stories__/chips.stories.js

```diff
@@ -92,30 +92,37 @@ export const Clickable = {
 };
 
 export const ChipsPalette = {
-  render: () => (
-    <Flex
-      style={{
-        width: "100%",
-        height: 300
-      }}
-      align={Flex.align.STRETCH}
-    >
-      {_chunk(Object.keys(Chips.colors), 7).map(colorChunk => {
-        return (
-          <Flex
-            direction={Flex.directions.COLUMN}
-            key={colorChunk}
-            justify={Flex.justify.SPACE_BETWEEN}
-            align={Flex.align.STRETCH}
-          >
-            {colorChunk.map(colorName => (
-              <Chips label={colorName} key={colorName} color={Chips.colors[colorName]} readOnly allowTextSelection />
-            ))}
-          </Flex>
-        );
-      })}
-    </Flex>
-  ),
+  render: () => {
+    const excludedColors = [Chips.colors.DARK_INDIGO, Chips.colors.BLACKISH];
```

Comment:
Did we decide to add a deprecate or TODO eventually?

Reply:
There's no intent (in the meantime at least) to disable the usage of it.
I'll ask design again to make sure

Reply:
@talkor my bad, added a comment, thanks!